### PR TITLE
stop always allowing today's date to be picked in datepicker

### PR DESCRIPTION
**Pull request by liampuk**
**Allowing 31 days to be selected from 15th June**

before:
Screenshot 2022-06-09 at 09 06 51

after:
Screenshot 2022-06-09 at 09 07 17

### CODE REVIEW

**Comment by fleonard** [comment]
Any extra context on this change? 
In most use cases we need to be able to select today as date (eg MTAs and new Inception Date in signup)

**Comment by liampuk** [comment]
> Any extra context on this change? In most use cases we need to be able to select today as date (eg MTAs and new Inception Date in signup)

This is normally really weird behaviour as we generally want to allow selectable days after a certain date, but today's date is always visible. Is this something we'd want to have as an optional prop?

**Comment by fleonard** [comment]
> > Any extra context on this change? In most use cases we need to be able to select today as date (eg MTAs and new Inception Date in signup)
> 
> This is normally really weird behaviour as we generally want to allow selectable days after a certain date, but today's date is always visible. Is this something we'd want to have as an optional prop?
> 
> @fleonard

@liampuk I guess it depends what we use as `fromDate`. We should hide today only if `fromDate` is not today otherwise should be included and active

**Comment by liampuk** [comment]
@fleonard yes that is how it should work, I'll check fromDate is inclusive of today's date in those places

**Review by mickaelzhang — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## [2.1.26] - 2022-06-09
+### Changed
+- Stop always allowing today's date to be picked
+
```
Comment (mickaelzhang) [inline CHANGELOG.md:4]: Need need to add a new version, you can add your change to `2.1.25`, it hasn't been released yet

**Pull request merged**